.travis.yml

```diff
@@ -18,11 +18,12 @@ before_install:
   - git clone https://github.com/nsls-ii/nsls2-ci ~/scripts
   - . ~/scripts/install-miniconda.sh
 
-  - git clone https://github.com/NSLS-II/metadataservice.git /home/travis/metadataservice
```

Comment:
Both parts of that -- your master branch being ahead of upstream and putting a user branch in the .travis.yml -- could be messy if we forget to change them back later. Let's get that MDS PR merged into upstream and leave this line as it was. Sound OK?